chore: explain performance improvements.

billy-the-fish opened the pull request:
Enable CAGGS to use MERGE for refresh operations rather than delete

billy-the-fish committed:
chore: explain performance improvements.

billy-the-fish committed:
chore: explain performance improvements.

billy-the-fish committed:
Merge branch 'latest' into 3513-docs-rfc-add-documentation-about-using-merge-for-continuous-aggregate-refresh

iroussos reviewed (commented):
Looks good to me @billy-the-fish, thank you for adding this to the docs!

Two comments:

(1) This is a parameter that you set and forget when you create continuous aggregates, not something that you need to set when you manually refresh a specific continuous aggregate.

Feels to me that this should be part of the [CREATE MATERIALIZED VIEW (Continuous Aggregate)](https://docs.timescale.com/api/latest/continuous-aggregates/create_materialized_view/#create-materialized-view-continuous-aggregate) page. What do you think?

(2) A small nitpick: 
The ["Configure database parameters" page](https://docs.timescale.com/use-timescale/latest/configuration/customize-configuration/) only focuses on the Timescale Cloud UI, not mentioning how a self hosted user running TimescaleDB can enable those parameters. But this is a core feature of TimescaleDB available to all users. 

Should we either add the command here or add a section on that page on how to enable parameters through psql?

billy-the-fish committed:
chore: updates on refresh.

billy-the-fish committed:
Merge branch 'latest' into 3513-docs-rfc-add-documentation-about-using-merge-for-continuous-aggregate-refresh

billy-the-fish commented:
Moved the config param to [CREATE MATERIALIZED VIEW (Continuous Aggregate)](https://docs.timescale.com/api/latest/continuous-aggregates/create_materialized_view/#create-materialized-view-continuous-aggregate) and linked to it from the refresh page, added the config to the TimescaleDB ref.
